workspace/python/ds-pack/2.1.0-base-python-3.10.13/Dockerfile

```diff
@@ -87,6 +87,16 @@ RUN curl -fsSL https://code-server.dev/install.sh \
     | sh -s -- --version 4.16.1 \
     && pip install --no-cache-dir ./codeserver-proxy
 
+# vscode cli
+RUN curl -Lk 'https://code.visualstudio.com/sha/download?build=stable&os=cli-alpine-x64' --output vscode_cli.tar.gz \
+    && tar -xf vscode_cli.tar.gz \
+    && cp ./code /usr/local/bin
+
+COPY vscode-tunnel vscode-tunnel
```

Comment:
Hey, did we try copying over these files to `/usr/local/bin` directly so that users could directly run `tunnel-start` or something like this?

Reply:
Or, we could even add this folder to `PATH` and user should be able to run it directly

Reply:
done.